tests/models/clip/test_modeling_clip.py

```diff
@@ -842,3 +842,33 @@ def test_inference(self):
         expected_logits = torch.tensor([[24.5701, 19.3049]], device=torch_device)
 
         self.assertTrue(torch.allclose(outputs.logits_per_image, expected_logits, atol=1e-3))
+
+    @slow
+    def test_inference_interpolate_pos_encoding(self):
+        # ViT models have an `interpolate_pos_encoding` argument in their forward method,
+        # allowing to interpolate the pre-trained position embeddings in order to use
+        # the model on higher resolutions. The DINO model by Facebook AI leverages this
+        # to visualize self-attention on higher resolution images.
+        model = CLIPModel.from_pretrained("openai/clip-vit-base-patch32").to(torch_device)
+
+        image_processor = CLIPProcessor.from_pretrained("openai/clip-vit-base-patch32", size=480)
```

Comment:
Three comments: 
* This is returning the processor, not the image_processor
* `size` should be a dictionary e.g. `size={"shortest_edge": 480}` here
* This won't test the interpolation, because the image processor crops after resizing. `crop_size` also has to be overriden

```suggestion
        processor = CLIPProcessor.from_pretrained("openai/clip-vit-base-patch32", size=480)
```